flasgger/base.py

```diff
@@ -33,15 +33,16 @@
 import jsonschema
 from mistune import markdown
 from .constants import OPTIONAL_FIELDS, OPTIONAL_OAS3_FIELDS
+from .utils import LazyString
 from .utils import extract_definitions
-from .utils import get_specs
 from .utils import get_schema_specs
+from .utils import get_specs
+from .utils import get_vendor_extension_fields
+from .utils import is_openapi3
 from .utils import parse_definition_docstring
 from .utils import parse_imports
-from .utils import get_vendor_extension_fields
-from .utils import validate
-from .utils import LazyString
 from .utils import swag_annotation
+from .utils import validate
```

Comment:
I just added the import `from .utils import is_openapi3` but isort sorted the existing import for me...